recipes/watchdog-gevent/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "watchdog-gevent" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/watchdog_gevent-{{ version }}.tar.gz
+  sha256: d19f1276a728dfb3ae3f1c8ada6e791f1726fd4336dff18b4ea35ed3ade35c6a
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - gevent >=1.1
+    - python
+    - watchdog >=0.8
+
+test:
+  imports:
+    - watchdog_gevent
+
+about:
+  home: The package home page
+  license: Apache Version 2.0
+  license_family: Apache
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
Can you add a comment with the link to your PR in the meta.yaml` here, to add the `license_file` to the sdist just as a reminder for the future once that PR is potential merged

Reply:
Added comment with link to issue.